app/controllers/api/users.php

```diff
@@ -7,7 +7,7 @@
 use Appwrite\Event\Delete;
 use Appwrite\Event\Event;
 use Appwrite\Network\Validator\Email;
-use Appwrite\Stats\Stats;
+use Appwrite\Usage\Stats;
```

Comment:
I don't think we use this namespace anywhere now, right? We can remove it from all controllers.

Reply:
will check and do